setup.py

```diff
@@ -129,8 +129,8 @@ def str_from_file(name):
 first_unsupported_version = "{}.{}".format(supported_python_versions[-1][0], supported_python_versions[-1][1] + 1)
 
 setup(name="opensearch-benchmark",
-      maintainer="Ian Hoang, Achit Ojha, Travis Benedict, Chase Engelbrecht, Govind Kamat, Phill Treddenick",
-      maintainer_email="hoangia@amazon.com, achiojha@amazon.com, benedtra@amazon.com, engechas@amazon.com, govkamat@amazon.com, treddeni@amazon.com",
+      maintainer="Ian Hoang, Govind Kamat",
+      maintainer_email="hoangia@amazon.com, govkamat@amazon.com",
```

Comment:
@gkamat do we want to keep these as is or change them to our Github affiliated emails?